blocks/property-search-bar/additional-filters.js

```diff
@@ -0,0 +1,218 @@
+import {
+  addRangeOption, EXTRA_FILTERS, formatInput, TOP_LEVEL_FILTERS, getConfig, processSearchType,
+} from './common-function.js';
+
+const SEARCH_TYPES = { ApplicationType: { label: 'Search Types', type: 'search-types' } };
+const FILTERS = { ...SEARCH_TYPES, ...TOP_LEVEL_FILTERS, ...EXTRA_FILTERS };
+
+function buildFilterSearchTypesElement() {
+  const config = getConfig('ApplicationType');
+  const columns = [[config[0], config[1]], [config[2], config[3]]];
+  let el;
+  let output = '<div class="column-2 flex-row">';
+
+  columns.forEach((column) => {
+    output += '<div class="column">';
+    column.forEach((value) => {
+      el = processSearchType(value);
+      el.querySelector('label').classList.add('text-up');
+      output += el.outerHTML;
+    });
+    output += '</div>';
+  });
+  output += '</div>';
+  return output;
+}
+
+export function buildFilterButtons() {
+  const buttons = ['apply', 'cancel', 'reset'];
+  const primary = ['apply'];
+  const wrapper = document.createElement('div');
+  wrapper.classList.add('filter-buttons', 'button-container', 'flex-row', 'vertical-center', 'hide');
+  let output = '';
+  buttons.forEach((button) => {
+    output += `<a title="${button}" rel="noopener" target="_blank" tabindex="" class="btn ${primary.includes(button) ? 'btn-primary' : 'btn-secondary'} center" role="button">
+            <span class="text-up ${primary.includes(button) && 'c-w'}">${button}</span>
+        </a>`;
+  });
+  wrapper.innerHTML = output;
+  return wrapper;
+}
+
+function buildPropertyColumn(labels = {}) {
+  let output = '';
+  Object.keys(labels).forEach((name) => {
+    output += `<button type="button" class="flex-row" value=${name}>
+                <svg role="presentation">
+                    <use xlink:href="/icons/icons.svg#${formatInput(labels[name])}"></use>
+                </svg>
+                <span class="ml-1">${labels[name]}</span>
+            </button>`;
+  });
+  return output;
+}
+
+function buildCheckBox(ariaLabel, label = '') {
```

Comment:
Is there a need for the ariaLabel to be different than the actual one?

Reply:
yeah.  We need to return empty labels in some cases. See Open houses checkbox